lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -127,6 +130,26 @@ def define_protocol(self):
         else:
             self.params['protocol'] = 'http'
 
+    def sanitize_keys(self, data):
+        if isinstance(data, dict):
+            items = {}
+            for k, v in data.items():
+                try:
+                    new = { self.key_map[k]: data[k] }
+                    items[self.key_map[k]] = self.sanitize_keys(data[k])
+                except KeyError:
+                    snake_k = re.sub('([a-z0-9])([A-Z])', r'\1_\2', k).lower()
+                    new = { snake_k: data[k] }
+                    items[snake_k] = self.sanitize_keys(data[k])
+            return items
+        elif isinstance(data, list):
```

Comment:
Instead of `list`, use `Sequence`. Similarly, replace `dict` above by `Mapping`. (You'll need `from ansible.module_utils.common._collections_compat import Mapping, Sequence` to be able to do that.)

Reply:
I haven't made this change yet, but I will.